app/routes/joblistings/components/JoblistingEditor.js

```diff
@@ -259,7 +260,10 @@ class JoblistingEditor extends Component<Props, State> {
                 message="Er du sikker på at du vil slette denne jobbannonsen?"
                 onConfirm={this.onDeleteJoblisting}
               >
-                <Button danger>Slett</Button>
+                <Button danger>
+                  <Icon name="trash" size={19} />
+                  Slett jobbannonse
+                </Button>
```

Comment:
This one is also not veritcally aligned with the button at it's side, and I also think this could benefit from having a "Cancel" button by its side.